src/main/java/com/hiqes/andele/Andele.java

```diff
@@ -208,8 +226,8 @@ public static void checkAndExecute(Activity activity, ProtectedAction action) {
      *                   the ActionCallback to execute when granted, etc.
      */
     @SuppressWarnings({"WeakerAccess", "unused"})
-    public static void checkAndExecute(AppCompatActivity activity, ProtectedAction action) {
-        checkAndExecute(new RequestOwnerAppCompatActivity(activity), action);
+    public static void checkAndExecute(android.support.v7.app.AppCompatActivity activity, ProtectedAction action) {
+        checkAndExecute(new RequestOwnerSupportAppCompatActivity(activity), action);
```

Comment:
It would be better to not change the type here since there could be codebases which already use it.  Introducing the new type for AndroidX above is ok, but keeping the existing type is needed for compatibility.

Reply:
Changes done as discussed.